doc/api/http.markdown

```diff
@@ -548,6 +548,11 @@ parameter is 511 (not 512).
 This function is asynchronous. The last parameter `callback` will be added as
 a listener for the `'listening'` event.  See also [`net.Server.listen(port)`][].
 
+### server.listening
+
+Returns a boolean telling if the server is whether or not listening for
```

Comment:
"Returns a Boolean indicating whether or not the server is listening for connections."